pkg/callout/callout.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package callout
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/absmach/supermq/pkg/errors"
+	svcerr "github.com/absmach/supermq/pkg/errors/service"
+)
+
+var errLimitExceeded = errors.New("limit exceeded")
+
+type Operations struct {
+	URLs            []string      `env:"URLS"             envDefault:"" envSeparator:","`
+	Method          string        `env:"METHOD"           envDefault:"POST"`
+	TLSVerification bool          `env:"TLS_VERIFICATION" envDefault:"true"`
+	Timeout         time.Duration `env:"TIMEOUT"          envDefault:"10s"`
+	CACert          string        `env:"CA_CERT"          envDefault:""`
+	Cert            string        `env:"CERT"             envDefault:""`
+	Key             string        `env:"KEY"              envDefault:""`
+	Permissions     []string      `env:"INVOKE_PERMISSIONS" envDefault:"" envSeparator:","`
```

Comment:
Please rename `Permissons` to `Operations` and  env name `INVOKE_PERMISSIONS` to `OPERATIONS`